### ISSUE TITLE
processArguments is run twice

### ISSUE THREAD

**Issue by chrisseaton** [jruby/jruby#3294]
```
	at org.jruby.RubyInstanceConfig.processArguments(RubyInstanceConfig.java:166)
	at org.jruby.Main.run(Main.java:224)
	at org.jruby.Main.main(Main.java:197)
```

```
	at org.jruby.RubyInstanceConfig.processArguments(RubyInstanceConfig.java:166)
	at org.jruby.Main.doProcessArguments(Main.java:468)
	at org.jruby.Main.internalRun(Main.java:257)
	at org.jruby.Main.run(Main.java:225)
	at org.jruby.Main.main(Main.java:197)
```

Is that the right behaviour?

**Comment by headius** [comment]
No, it's not!

**Comment by headius** [comment]
Fixed for 1.7.23 and 9.0.2.0.
